packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -68,7 +70,7 @@ class OutlineButton extends StatefulWidget {
     this.highlightedBorderColor,
     this.padding,
     this.shape,
-    this.clipBehavior = Clip.none,
+    this.clipBehavior = ui.defaultClipBehavior, // ignore: deprecated_member_use,
```

Comment:
You provide the same default value for clipBehavior in RawMaterialButton, so there's no need to specify it here.

Reply:
same as above